### PR TITLE
 Remove @asyncio.coroutine + 3.4 Support Fixes #26 

### LINKED ISSUE #26: Remove @asyncio.coroutine + 3.4 Support
What are the thoughts of dropping 3.4 support and mainly `@asyncio.coroutine` decorator now we're in 2020 and many versions of Python ahead. This would be replaced with `async def` coroutine creation.

3.4 is not the default anymore on most popular Linux distros too,

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@mosquito thoughts on enabling black/flake8 etc with pre-commit?

**Comment by mosquito** [comment]
0.8.1 already on pypi

**Comment by mosquito** [comment]
Thank you

**Comment by graingert** [comment]
What happend to 0.8.0? Also could you update the git tags?

File: aiohttp_xmlrpc/handler.py [hunk 1]
```diff
@@ -94,7 +92,7 @@ def _handle(self):
         else:
             kwargs = {}
 
-        result = yield from asyncio.coroutine(method)(*args, **kwargs)
+        result = await asyncio.coroutine(method)(*args, **kwargs)
```
Comment (graingert) [inline aiohttp_xmlrpc/handler.py]: Oh woops this isn't right @mosquito

**Comment by mosquito** [comment]
0.8.0 was wrong release about a year ago. Pypi disallow upload 0.8.0 with different content.

File: aiohttp_xmlrpc/handler.py (hunk 1 above)
Comment (mosquito) [reply aiohttp_xmlrpc/handler.py]: Yes but we have to replace it like this https://github.com/mosquito/aiormq/blob/master/aiormq/tools.py#L21

**Pull request merged**